src/app/components/Modals/UnbondModal.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  DialogHeader,
+  Text,
+} from "@babylonlabs-io/bbn-core-ui";
+
+interface UnbondModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  onProceed: () => void;
+}
+
+export const UnbondModal = ({
```

Comment:
In #373 we use `V2` in naming
Should we use it here? Or should we remove it from #373?